api/V1/Device.js

```diff
@@ -279,4 +279,51 @@ module.exports = function(app, baseUrl) {
       });
     })
   );
+
+  /**
+   * @api {post} /api/v1/devices/query Querys devices by group:name.
+   * @apiName query
+   * @apiGroup Device
+   * @apiDescription This call is to query all devices by groups or devicenames
+   *
+   * @apiUse V1DeviceAuthorizationHeader
+   *
+   * @apiParam {JSON} devices array of device obejcts [{devicename:"<name>",groupname:"<group>"}].
```

Comment:
objects

Reply:
You can use `@apiParamExample` for the example here (so that it's formatted nicely)